packages/web-pkg/src/components/SideBar/SideBar.vue

```diff
@@ -203,11 +204,21 @@ export default defineComponent({
       return $gettext('Back to main panels')
     })
 
-    const fullWidthSideBar = computed(() => window.innerWidth <= 1024)
+    const windowWidth = ref(window.innerWidth)
+
+    const fullWidthSideBar = computed(() => unref(windowWidth) <= 960)
     const backgroundContentEl = computed(() => {
       return unref(appSideBar)?.parentElement?.querySelector('div') as HTMLElement
     })
 
+    const onResize = () => {
```

Comment:
maybe we can think of debouncing the function.

Reply:
That might cause again "laggy" reactivity. Resize does not happen that often so I think it is fine not to have it debounced.